tests/ibge/test_uf.py

```diff
@@ -1,6 +1,9 @@
 from unittest import TestCase
 
-from brutils.ibge.uf import convert_code_to_uf
+from brutils.ibge.uf import (
+    convert_code_to_uf, 
```

Comment:
Espaço extra está causando erro de lint:

```suggestion
    convert_code_to_uf,
```